lib/convertkit/resources/subscribers.rb

```diff
@@ -0,0 +1,67 @@
+module ConvertKit
+  module Resources
+    class Subscribers
+      PATH = 'subscribers'.freeze
+
+      def initialize(client)
+        @client = client
+      end
+
+      # Returns a list of subscribers for the account.
+      # See https://developers.convertkit.com/#list-subscribers for details
+      # @param [Hash] options
+      # @option options [Integer] :page The page of results to return.
+      # @option options [Integer] :from Filter subscribers added on or after date
+      # @option options [Integer] :to Filter subscribers added on or before date
+      # @option options [String] :updated_from Filter subscribers updated after date
+      # @option options [String] :updated_to Filter subscribers updated before date
+      # @option options [String] :sort_order Sort order for results(`asc` or `desc`)
+      # @option options [String] :sort_field Field to sort results by
+      # @option options [String] :email_address Search subscribers by email address
+      def get_subscribers(options = {})
```

Comment:
Good question. The only default value mention in the API documentation is for the `page` attribute. I can add that in the comment.

Reply:
Do we know what the defaults will be? Not sure if we've tested the endpoints against the real one yet. This can be updated later as well if we haven't. Not blocking this PR